src/data/services/LmsApiService.js

```diff
@@ -384,6 +385,57 @@ class LmsApiService {
     const url = `${LmsApiService.baseUrl}/enterprise/api/v1/analytics-summary/${enterpriseUUID}`;
     return LmsApiService.apiClient().post(url, formData);
   }
+
+  static updateUserActiveEnterprise = (enterpriseId) => {
+    const url = `${configuration.LMS_BASE_URL}/enterprise/select/active/`;
+    const formData = new FormData();
+    formData.append('enterprise', enterpriseId);
+
+    return LmsApiService.apiClient().post(
+      url,
+      formData,
+    );
+  };
+
+  static loginRefresh = async () => {
+    const loginRefreshUrl = `${configuration.LMS_BASE_URL}/login_refresh/`;
+
+    try {
+      return LmsApiService.apiClient().post(loginRefreshUrl);
+    } catch (error) {
+      const isUserUnauthenticated = error.response?.status === 401;
+      if (isUserUnauthenticated) {
+        // Clean up the cookie if it exists to eliminate any situation
+        // where the cookie is not expired but the jwt is expired.
+        const cookies = new Cookies();
+        cookies.remove(configuration.ACCESS_TOKEN_COOKIE_NAME);
+      }
+      return Promise.resolve();
+    }
+  };
+
+  static fetchEnterpriseLearnerData(options) {
```

Comment:
Might want to have test coverage on these new service methods to ensure the correct API endpoints with expected query params are called.

Reply:
added!